### ISSUE TITLE
what is this?

### ISSUE THREAD

**Issue by AsynchronousAI** [bamyt212/Headshot#1]
# What is this?

- gitignore is massive
- description and README.md is just code? (Lua I think)

**Comment by Cotton45013** [comment]
67.231.152.131

**Comment by AsynchronousAI** [comment]
yeah that IP address doesn't work, it is a Proofpoint server. located at 37.7510 / 37°45′3″ N, -97.8220 / 97°49′19″ W

**Comment by AsynchronousAI** [comment]
mx0b-0030e801.pphosted.com

**Comment by Cotton45013** [comment]
Amy banker hacking evidence trace
